components/gitpod-db/src/typeorm/entity/db-terms-acceptance-entry.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2020 TypeFox GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { Column, Entity, PrimaryColumn } from "typeorm";
+import { TermsAcceptanceEntry } from "@gitpod/gitpod-protocol";
+
+@Entity()
```

Comment:
A short comment about how the DB is used would be nice. E.g.: "single entry per user which is overriden/updated over time"